.gitattributes

```diff
@@ -1,2 +1,4 @@
 * text=auto eol=lf
 *.bat text eol=crlf
+# GitHub syntax highlighting
+pixi.lock linguist-language=YAML linguist-generated=true
```

Comment:
It tells GitHub how to render diffs? We don't need it anymore.

Reply:
I undid these changes.